docker/openemr/5.0.2/openemr.conf

```diff
@@ -8,64 +9,68 @@ LoadModule rewrite_module modules/mod_rewrite.so
 	# However, you must set it for any further virtual host explicitly.
 	#ServerName www.example.com
 
+	## Security Options
+    Protocols http/1.1
+    # Removes revealing HTTP Headers
+    ServerSignature off
+    ServerTokens Prod
+    TraceEnable Off
+    FileETag None
+    Header unset ETag
+    # Set HSTS and X-XSS protection
+    Header set Strict-Transport-Security "max-age=31536000; includeSubDomains; preload"
+    Header set X-XSS-Protection "1; mode=block"
 	ServerAdmin webmaster@localhost
+	# Narrow document root
 	DocumentRoot /var/www/localhost/htdocs/openemr
-	<Directory /var/www/localhost/htdocs/openemr>
-		Options Indexes FollowSymLinks
-	        AllowOverride None
-		Require all granted
-	</Directory>
+    ErrorLog ${APACHE_LOG_DIR}/error.log
+    CustomLog ${APACHE_LOG_DIR}/access.log combined
 
- <Directory "/var/www/localhost/htdocs/openemr">
-      AllowOverride FileInfo
-  </Directory>
-  <Directory "/var/www/localhost/htdocs/openemr/sites">
-      AllowOverride None
-  </Directory>
-  <Directory "/var/www/localhost/htdocs/openemr/sites/*/documents">
-      order deny,allow
-      Deny from all
-  </Directory>
-  <Directory "/var/www/localhost/htdocs/openemr/sites/*/edi">
-      order deny,allow
-      Deny from all
-  </Directory>
-  <Directory "/var/www/localhost/htdocs/openemr/sites/*/era">
-      order deny,allow
-      Deny from all
-  </Directory>
+    <Directory /var/www/localhost/htdocs/openemr>
+        # Only allow these HTTP Methods
+        AllowMethods GET POST HEAD OPTIONS
+        # No indexes anywhere
+        Options -Indexes
+        Require all granted
+    </Directory>
 
-	# Available loglevels: trace8, ..., trace1, debug, info, notice, warn,
-	# error, crit, alert, emerg.
-	# It is also possible to configure the loglevel for particular
-	# modules, e.g.
-	#LogLevel info ssl:warn
-	ErrorLog ${APACHE_LOG_DIR}/error.log
-	CustomLog ${APACHE_LOG_DIR}/access.log combined
+    <Directory "/var/www/localhost/htdocs/openemr">
+         # Keeping this it might do something important
+         AllowOverride FileInfo
```

Comment:
This is needed for Zend, Patient Portal, and the API to work (since they all use a .htaccess file for redirects etc.).

Reply:
Got it thank you very much letting me know what this does